### ISSUE TITLE
Fine-tuned OmniVoice sometimes generates artifact / non-speech audio during inference

### ISSUE THREAD

**Issue by sara-mohammadi2001** [k2-fsa/OmniVoice#193]
### Checks

- [x] This template is only for research question, not usage problems, feature requests or bug reports.
- [x] I have thoroughly reviewed the project documentation and read the related paper(s).
- [x] I have searched for existing issues, including closed ones, no similar questions.
- [x] I am using English to submit this issue to facilitate community communication.

### Finetuning Details
I fine-tuned the pretrained OmniVoice model on my own dataset. I did not make major changes to the default configs, except changing the number of training steps.

Dataset details:

- Number of hours: 2600 hours
- Number of speakers: +10
- Text language: persian

Training config changes:

```json

{

  "init_from_checkpoint": "k2-fsa/OmniVoice",
  "steps": "100000",
  "learning_rate": "1e-5",
  "attention_type": "sdpa",

}
```

# Problem
After fine-tuning, inference works correctly for many samples, but in some cases the generated audio contains strong artifacts. The waveform is generated successfully and no Python error is thrown, but part of the output contains distortion, crackling, meaningless sound, or unusable audio. This does not happen for all inputs. It happens only for some texts / samples.

I already checked issue #144: Crackling or unintelligible audio for long or awkward text inputs. That issue seems related because it also mentions bad audio artifacts / unintelligible generated speech. However, in my case the problem appears after fine-tuning the OmniVoice model on my own dataset, with almost default configs except for changing the number of training steps.

I would like to know whether this is expected behavior after fine-tuning, whether it can be caused by overtraining or inference parameters, and what the recommended fix is.

**Comment by zhu-han** [comment]
If you observe degraded audio quality after fine-tuning, it is most likely caused by low-quality fine-tuning data or improper fine-tuning hyperparameters (e.g., learning rate, training steps). So the first step is to check whether your training data is clean enough (e.g., audio quality, transcription accuracy). If you are sure that your data is suitable for TTS training, you can try tuning the learning rate (smaller) and check the intermediate checkpoints to see when it goes wrong.
